src/main/java/org/miracum/etl/fhirtoomop/model/omop/Concept.java

```diff
@@ -79,7 +79,7 @@ public String getLabel() {
   }
 
   public enum Vocabulary {
-    ICD10GM("ICD10GM"),
+    ICD10GM("ICD10"),
```

Comment:
Should the `enum` "key" change to match?

Reply:
Will be in the "vocabulary" update PR.